contracts/mirror_mint/src/contract.rs

```diff
@@ -243,6 +248,8 @@ pub fn try_update_asset<S: Storage, A: Api, Q: Querier>(
         asset.min_collateral_ratio = min_collateral_ratio;
     }
 
+    asset.mint_end = mint_end;
```

Comment:
We must let the devs know about mint_end always exist when updating other values. If you can remove this, it looks better